### DISCUSSION TITLE
Seeking Advice on Best Approach to Integrate BIRT with Spring WebFlux Project

### DISCUSSION THREAD

**Discussion by geo-george-k** [eclipse-birt/birt#1623] [Q&A]
I'm currently working on **_integrating BIRT_** reporting functionality into our **_Spring WebFlux_** project, and I'm seeking advice from the community on the best approach to accomplish this task effectively. While I've explored various options, I'm unsure about the optimal method. And there is **_no consistently up-to-date maven dependency or documentation or references_**. I'd greatly appreciate any insights, recommendations, or experiences from the community regarding the best approach to integrate BIRT with our Spring WebFlux project. Whether it's leveraging specific libraries, frameworks, or architectural patterns, I'm open to exploring all possibilities.

Thank you in advance for your help and guidance!

**Comment by speckyspooky** [comment]
Please take a look into #1602 there are some links and information from today.
